paimon-common/src/main/java/org/apache/paimon/predicate/CompoundPredicate.java

```diff
@@ -52,6 +53,11 @@ public boolean test(Object[] values) {
         return function.test(values, children);
     }
 
+    @Override
+    public boolean test(InternalRow row) {
+        return false;
```

Comment:
this should be `funciton.test(row, children)` ?

Reply:
you are right